index.js

```diff
@@ -35,20 +35,27 @@ module.exports = (string, options) => {
 
 	options = Object.assign({
 		separator: '-',
-		customReplacements: []
+		customReplacements: [],
+		lowerCase: true
```

Comment:
The option should be named `lowercase`.

See: https://english.stackexchange.com/questions/59409/lowercase-lower-case-or-lower-case